tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -26,21 +39,106 @@ public void run() {
     @Override
     public void process(final Socket connection) {
         try (final var inputStream = connection.getInputStream();
+             final BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
              final var outputStream = connection.getOutputStream()) {
 
-            final var responseBody = "Hello world!";
+            final String requestUri = parseUri(reader);
+            log.info("request uri : {}", requestUri);
 
-            final var response = String.join("\r\n",
-                    "HTTP/1.1 200 OK ",
-                    "Content-Type: text/html;charset=utf-8 ",
-                    "Content-Length: " + responseBody.getBytes().length + " ",
-                    "",
-                    responseBody);
+            if (reader.readLine() == null) {
+                return;
+            }
+
+            final Map<String, String> header = parseHeader(reader);
+            final String response = getResponse(requestUri, header).toText();
 
             outputStream.write(response.getBytes());
             outputStream.flush();
-        } catch (IOException | UncheckedServletException e) {
+        } catch (IOException | UncheckedServletException | URISyntaxException e) {
             log.error(e.getMessage(), e);
         }
     }
+
+    private String parseUri(final BufferedReader reader) throws IOException {
+        return reader.readLine().split(" ")[1];
+    }
+
+    private Map<String, String> parseHeader(final BufferedReader reader) throws IOException {
+        final Map<String, String> header = new HashMap<>();
+        String line = reader.readLine();
+        while (!"".equals(line)) {
+            final String[] splitLine = line.split(": ", 2);
+            header.put(splitLine[0], splitLine[1]);
+            line = reader.readLine();
+        }
```

Comment:
```suggestion
        while (!"".equals(line)) {
            final String[] splitLine = line.split(": ", 2);
            header.put(splitLine[KEY], splitLine[VALUE]);
            line = reader.readLine();
        }
```

상수로 만들어두면 코드를 이해하기 쉬울 것 같아요!